Docs/Book/Cookbook_v2.rst

```diff
@@ -0,0 +1,1017 @@
+
+What gets included?
+=====================
+
+The examples included come from various online sources such as blogs, shared gists, and 
+the RDKit mailing lists. Generally, only minimal editing is added to the example code/notes for 
+formatting consistency and to incorporate the doctests. We have made a conscious effort 
+to appropriately credit the original source and authors. One of the first priorities of this
+document is to compile useful **short** examples shared on the RDKit mailing lists, as 
+these can be difficult to discover. It will take some time, but we hope to expand this 
+document into 100s of examples. As the document grows, it may make sense to prioritize 
+examples included in the RDKit Cookbook v2 based on community demand.
+
+Feedback and Contributing
+==========================
+
+If you have suggestions for how to improve the Cookbook v2 and/or examples you would like 
+included, please contribute directly in the source document (the .rst file).
+Alternatively, you can also send Cookbook revisions and addition requests to the mailing list:
+<rdkit-discuss@lists.sourceforge.net> (you will need to subscribe first).
+
+.. note::
+
+   The Index ID# (e.g., **RDKitCB_##**) is simply a way to track Cookbook entries and image file names. 
+   New Cookbook additions are sequentially index numbered, regardless of where they are placed 
+   within the document. As such, for reference, the next Cookbook entry is **RDKitCB_20**.
+
+Drawing Molecules (in a Jupyter Environment)
+**********************************************
+
+Include an Atom Index
+======================
+
+| **Author:** Takayuki Serizawa
+| **Source:** `<https://iwatobipen.wordpress.com/2017/02/25/draw-molecule-with-atom-index-in-rdkit/>`_
+| **Index ID#:** RDKitCB_0
+| **Summary:** Draw a molecule with atom index numbers.
+
+.. testcode::
+
+   from rdkit import Chem
+   from rdkit.Chem.Draw import IPythonConsole
+   from rdkit.Chem import Draw
+   IPythonConsole.ipython_useSVG=False
+   import rdkit
+
+.. testcode::
+  
+   def mol_with_atom_index(mol):
+       atoms = mol.GetNumAtoms()
+       for idx in range(atoms):
+           atom = mol.GetAtomWithIdx(idx)
+           atom.SetProp('molAtomMapNumber', str(idx))  
+       return mol
+
+.. testcode::
+
+   # Test in a kinase inhibitor
+   mol = Chem.MolFromSmiles("C1CC2=C3C(=CC=C2)C(=CN3C1)[C@H]4[C@@H](C(=O)NC4=O)C5=CNC6=CC=CC=C65")
+   # Default
+   mol
+   
+.. image:: images/RDKitCB_0_im0.png
+
+.. testcode::
+  
+   # With atom index
+   mol_with_atom_index(mol)
+   
+.. image:: images/RDKitCB_0_im1.png
+
+Black and White Molecules
+==========================
+
+| **Author:** Greg Landrum
+| **Source:** `<https://gist.github.com/greglandrum/d85d5693e57c306e30057ec4d4d11342>`_
+| **Index ID#:** RDKitCB_1
+| **Summary:** Draw a molecule in black and white.
+
+.. testcode::
+
+   from rdkit import Chem
+   from rdkit.Chem.Draw import IPythonConsole
+   from rdkit.Chem import Draw
+   import rdkit
```

Comment:
you don't need the `import rdkit`
I include that in notebooks so that I can do `print(rdkit.__version__)`, but that's not needed here